test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -87,6 +91,31 @@ namespace iroha {
       std::string block_store_path = "/tmp/block_store";
     };
 
+    TEST_F(AmetsuchiTest, GetBlocksCompletedWhenCalled) {
+      // Commit block => get block => observable completed
+      auto storage =
+          StorageImpl::create(block_store_path, redishost_, redisport_, pgopt_);
+      ASSERT_TRUE(storage);
+
+      model::Block block;
+      block.height = 1;
+
+      auto ms = storage->createMutableStorage();
+      ms->apply(block, [](const auto &blk, auto &executor, auto &query,
+                          const auto &top_hash) { return true; });
```

Comment:
The block created before does not contain any commands. `apply()` is called here to append the block only.